utils/data_loader_fv3gfs.py

```diff
@@ -110,13 +110,13 @@ def _load_stats_data(self):
         # just used for multistep validation
         logging.info(f"Opening time mean stats data at {self.params.time_means_path}")
         in_, out_ = load_arrays_from_netcdf(self.params.time_means_path, self.in_names, self.out_names)
```

Comment:
Agreed it would make the code more readable. For most of our analysis work, we use xarray which includes named dimensions and meaningful coordinates, and helps a lot with these issues. But for performance reasons I don't think we want to introduce that dependency here.

We could consider "named tensors", which are a newish feature in pytorch: https://pytorch.org/docs/stable/named_tensor.html.

Reply:
Named tensors would great but maybe a bit too much for this change / at this stage? I was thinking something super simple, e.g. introducing a few unused variables just so that whoever comes back to this later won't have to rediscover the order of the dims -- unless this is something which is already standard in weather/climate models.

Reply:
Yeah, don't really think we should used name tensors here, but more as something to consider in future.

There is a conventional dimension order (time, height, latitude, longitude) although we can't assume it to hold in all cases. For example see this convention standard: https://ferret.pmel.noaa.gov/Ferret/documentation/coards-netcdf-conventions. In particular FourCastNet doesn't have a notion of 'height' but does have a 'channel' dim, and in some cases a 'batch' one too.